core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/IntelliJFactory.scala

```diff
@@ -19,7 +20,7 @@ final class IntelliJFactory(
   def create(version: IntelliJVersion, plugins: Seq[Plugin]): InstalledIntelliJ = {
     val root = createInstanceDirectory(version)
 
-    val allPlugins = InternalPlugins.all ++ plugins
+    val allPlugins = probePluginForIntelliJ(version) +: plugins
```

Comment:
If we would keep the `InternalPlugins` it would be more clear that function comes from outside rather than the base trait (specifically it is more important for the PluginExtension traits.